Jenkinsfile

```diff
@@ -28,8 +36,8 @@ node {
 			targetfiles="$(find releng -type f -iname '*.target')"
 			for targetfile in $targetfiles
 			do
-				echo "Redirecting target platforms in $targetfile to $JENKINS_URL"
```

Comment:
ia this change intentional?

Reply:
To be honest, not absolutly sure. Done this together with @kthoms and have not a full understanding of this detail.